### DISCUSSION TITLE
[question/discussion] singular spectrum transformation vs. matrix profile

### DISCUSSION THREAD

**Discussion by geoHeil** [TDAmeritrade/stumpy#264] [Q&A]
Does anyone have experience with singular spectrum transformation and in particular also a comparison with the matrix profile?

**Comment by seanlaw** [comment]
For consistency in Github's search capability, please try to add a space in between "matrix" and "profile". This will make it easier for people who are also looking for the "matrix profile" topic or keywords.

To answer your question, I do not have any experience but others may

**Comment by seanlaw** [comment]
It may be useful to provide some references to "singular spectrum transformation" and you may want to consider cross posting to [Cross Validated](https://stats.stackexchange.com/)

**Comment by geoHeil** [comment]
will do so and close it here for now: https://stats.stackexchange.com/questions/490807/matrix-profile-vs-singular-spectrum-transformation
